apps/tsilhqotin-language-hub/src/Components/Pages/Song/Song.tsx

```diff
@@ -31,13 +33,24 @@ export function Song(props: SongsDetailComponentProps) {
                     <h1 id="pageTitle"> {`${title || titleEnglish}`}</h1>
```

Comment:
Let's build that helper (presentational pipe) for the bilingual text presenter as we had in Angular previously.

Reply:
Done.